altair/examples/normed_parallel_coordinates.py

```diff
@@ -0,0 +1,36 @@
+"""
+Normalized Parallel Coordinates Example
+----------------------------
```

Comment:
Line needs to be the same length as the text above, or the sphinx RST->html tool will complain.

Reply:
yep, please see the revised code